gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -245,6 +245,10 @@ spec:
             value: {{ $val }}
           {{- end }}
           {{- end }}
+          {{- if and (not $gateway.env.ISTIO_META_NETWORK) .Values.global.network }}
+          - name: ISTIO_META_NETWORK
+            value: {{ .Values.global.network }}
+          {{- end }}
```

Comment:
@rshriram, is there ever any reason the value of ISTIO_META_NETWORK should differ between the sidecar and ingress-gateway proxies in a mesh?

Reply:
no.. the only usecase is the egress gateway

Reply:
This PR should be good then if they never differ. One less option to configure.

I don't see ISTIO_META_NETWORK used for egress but I do see ISTIO_META_REQUESTED_NETWORK_VIEW.